src/geovista/geodesic.py

```diff
@@ -630,11 +631,12 @@ def enclosed(
         tolerance : float, optional
             The tolerance on the intersection operation with the `surface`,
             expressed as a fraction of the diagonal of the bounding-box.
-            Defaults to :data:`BBOX_TOLERANCE`.
+            Defaults to :data:`BBOX_TOLERANCE`. See PyVista's
+            :meth:`~pyvista.DataSetFilters.select_enclosed_points` for more.
         outside : bool, optional
             By default, select those points of the `surface` that are inside
             the bounding-box. Otherwise, select those points that are outside
-            the bounding-box. Defaults to ``False``.
```

Comment:
I'd prefer to keep this please.

I've adopted the standard of explicitly stating the default value in the docstrings throughout, ta.

Reply:
Sorry this was a mistake when I backed out an edit I made.

Reply:
4f5a5cc4098eb1cba76e857142245953f6ec7cbf